### ISSUE TITLE
Add option to open Playnite directly with a single left-click on tray icon

### ISSUE THREAD

**Issue by ATNewHope** [JosefNemec/Playnite#2784]
### Feature description

Now, if I left-click once on tray icon, a menu pop up.
But I actually want Playnite pop up, and if I want a menu, I could use Right click.
So maybe an option about this could be added.
Thanks in advance!

**Comment by JosefNemec** [comment]
You double click to open Playnite directly.

**Comment by ATNewHope** [comment]
> You double click to open Playnite directly.

Yeah, you're right, but when I'm using touch screen it will be more easy if only a single click could do it.

**Comment by 06000208** [comment]
I would prefer this as a mouse user, and found this issue looking for a way to configure the tray icon, as I enjoy consistency amongst my apps.

Apps such as discord, epic games, and copyq open their main app window on left click, while sharex is an example of an app that can be configured to behave like this.

Meanwhile, steam is an example of an app that acts the same as playnite, so I wouldn't be surprised if there were others.

There's also the observation that opening the context menu with right click in both steam and playnite is faster, as they don't have to wait for a double click to determine what to do.
